packages/rapidpro/src/Adaptor.js

```diff
@@ -104,41 +103,45 @@ export function upsertContact(params, callback) {
       headers: { Authorization: `Token ${token}` },
     };
```

Comment:
I can't comment in the right place because it's outside the diff - but this `config` object, which used to be used by axios, can now be removed. It has no value.